salt/proxy/dummy.py

```diff
@@ -35,16 +38,14 @@ def __virtual__():
 
 
 def _save_state(details):
-    pck = open(FILENAME, 'wb')  # pylint: disable=W8470
-    pickle.dump(details, pck)
-    pck.close()
+    with salt.utils.fopen(FILENAME, 'wb') as pck:
```

Comment:
This line needs to use the new path `salt.utils.files.fopen` instead of `salt.utils.fopen`.